pull_request_template.md

```diff
@@ -0,0 +1,52 @@
+## Purpose
+> Describe the problems, issues, or needs driving this feature/fix and include links to related issues in the following format: Resolves issue1, issue2, etc.

```

Comment:
```suggestion
> Describe the problems, issues, or needs driving this feature/fix and include links to related issues in the following format: Resolves issue1, issue2, etc.

```
Why is this file committed?

Reply:
This is a file that comes with the initial commit when creating the repo.